news/2021-04-27-results-a-labour-of-love.md

```diff
@@ -0,0 +1,101 @@
+
+### Overall winner
+
+Taking the breathe away from mappers and musicians across the board, team *extremely valid* wears the heart-shaped crown with their submission [**Raphlesia & BilliumMoto - My Love**](https://osu.ppy.sh/beatmapsets/1388906#osu/2868388)!
+
+![Overall runner-up badge](/wiki/shared/news/2021-04-27-results-a-labour-of-love/overall.jpeg)
+
+Yep, that's right. This entry received #1 in the musician-focused panel, the mapping panel, and the overall panel. With just a few more votes they could've even taken the community vote — insane, right?
+
+This remix by [Shinwoir](https://osu.ppy.sh/users/8984574) (a.k.a. Raphlesia) and [BilliumMoto](https://osu.ppy.sh/users/3862471) is practically perfect for a rhythm game setting, and who would've expected less from a combo involving the last [osu! remix contest](https://osu.ppy.sh/community/contests/1) winner? [Mao](https://osu.ppy.sh/users/2204515) / [DeviousPanda](https://osu.ppy.sh/users/4966334) / [Realazy](https://osu.ppy.sh/users/918297) took full advantage of this perfect track too, crafting a 6-difficulty spread that appeals to players on every level with stellar performances from everyone involved. While this entry's storyboard was a bit lackluster compared to the other entries, this one clearly focused on the other aspects and managed to make it work anyway.
+
+Given their landslide of a win on the judging side of things, it's more than fitting to reward everyone on team *extremely valid* with the overall winner profile badge!
+
+![Overall winner badge](/wiki/shared/news/2021-04-27-results-a-labour-of-love/alol_overallwinner@2x.png)
+
+On top of this, each member of the team has earned a forum title to honour their win:
+
+- [BilliumMoto](https://osu.ppy.sh/users/3862471): osu!mixer II
+- [Shinwoir](https://osu.ppy.sh/users/8984574): osu!mixer
+- [Mao](https://osu.ppy.sh/users/2204515): Elite Mapper II
+- [DeviousPanda](https://osu.ppy.sh/users/4966334): Elite Mapper
+- [Realazy](https://osu.ppy.sh/users/918297): Elite Mapper
+
+Given [BilliumMoto](https://osu.ppy.sh/users/3862471) and [Mao](https://osu.ppy.sh/users/2204515) already earned these user titles for different contests, they've been granted the *II* suffix. Double congrats to them!
+
+## Conclusion
+
+Congratulations again to the winners and thanks to everyone who participated in A Labour of Love!
+
+We hope this contest has been as fun for you as it has been for us. Community members joining together to produce something cool has been the spirit of osu! since the very beginning, and it's awesome to see that feeling shine through in every submission (...except maybe [this one](https://osu.ppy.sh/beatmapsets/1348553), sorry team Down).
```

Comment:
```suggestion
We hope this contest has been as fun for you as it has been for us. Community members joining together to produce something cool has been the spirit of osu! since the very beginning, and it's awesome to see that feeling shine through in every submission (...except maybe [this one](https://osu.ppy.sh/beatmapsets/1348553), sorry team *Down*).
```